tket/src/Circuit/ThreeQubitConversion.cpp

```diff
@@ -154,8 +154,8 @@ static std::pair<Circuit, Complex> two_qubit_plex(
 
   // We try conjugating the L and R circuits to see if we can reduce CX count.
   std::optional<Circuit> best_circ;
-  Complex best_z0 = 0;
-  unsigned best_n_cx = 0;
+  Complex best_z0 = 1.;
+  unsigned best_n_cx = UINT_MAX;
```

Comment:
```suggestion
  unsigned best_n_cx = std::numeric_limits<unsigned>::max();
```
Probably more portable.

Reply:
Sorry, do you mind if I just leave them uninitialised as before?

Reply:
> Sorry, do you mind if I just leave them uninitialised as before? (didn't mean to push this change).

I guess you do need to initialize as the compiler is not clever enough to see that it makes no difference.

Reply:
Ah yes, sorry. I am distracted.

Reply:
In the end I suggest initialising to `std::nullopt`. It's less memory efficient but makes my intentions explicit.